app.js

```diff
@@ -1,21 +1,19 @@
 const express = require('express');
 const app = express();
-const { fetchCategories, fetchReviews } = require('./controllers/get-controllers');
-const { handleServerErrors } = require('./error-handlers');
+const { fetchCategories, fetchReviews, fetchReview } = require('./controllers/get-controllers');
+const { handlesServerErrors, handlesInvalidPath, handles404Errors, handles400Errors } = require('./error-handlers');
 
 // app.use(express.json());
```

Comment:
We can get rid of this for now as it is not needed and we don't want comments left when submitting PRs :)